cli/command/image/build.go

```diff
@@ -184,12 +184,6 @@ func runBuild(dockerCli command.Cli, options buildOptions) error {
 		remote        string
 	)
 
-	if !options.quiet && dockerCli.ServerInfo().OSType != "windows" {
```

Comment:
Reminds me; I didn't try if the other location takes `options.quiet` into account

Reply:
No it's only taken into account in build pkg that's why it was easier to add the message here. We would need to parse the command and check if quiet is enabled and a build command is being used. I don't think we should do that their imo.